indico/modules/events/editing/client/js/management/editable_type/NextEditable.jsx

```diff
@@ -0,0 +1,273 @@
+          extensions: _.pickBy(filters, x => Array.isArray(x)),
+          has_files: _.pickBy(filters, x => !Array.isArray(x)),
+        });
+      } catch (e) {
+        handleAxiosError(e);
+        setLoading(false);
+        return;
+      }
+      setFilteredEditables(camelizeKeys(response.data));
+      setLoading(false);
+    })();
+  }, [eventId, editableType, filters]);
+
+  const updatePresenceFilter = (id, present) => {
+    if (filters[id] === present) {
+      setFilters(_.omit(filters, id));
+    } else {
+      setFilters({...filters, [id]: present});
+    }
+  };
+
+  const updateExtensionFilter = (id, extension) => {
+    const typeFilter = filters[id];
+    if (!Array.isArray(typeFilter)) {
+      // switching from no filter or presence filter to extension filter
+      setFilters({...filters, [id]: [extension]});
+    } else if (!typeFilter.includes(extension)) {
+      // adding previously-eselected extension
+      setFilters({...filters, [id]: [...typeFilter, extension]});
+    } else if (typeFilter.length === 1) {
+      // deselecting last extension
+      setFilters(_.omit(filters, id));
+    } else {
+      // deselecting some extension
+      setFilters({...filters, [id]: _.without(typeFilter, extension)});
+    }
+  };
+
+  const handleAssign = async () => {
+    try {
+      await indicoAxios.put(
+        assignMyselfURL({
+          confId: eventId,
+          contrib_id: selectedEditable.contributionId,
+          type: editableType,
+        })
+      );
+    } catch (error) {
+      handleAxiosError(error);
+      return;
+    }
+    location.href = selectedEditable.timelineURL;
+  };
+
+  const titleByType = {
+    paper: Translate.string('Get next paper'),
+    poster: Translate.string('Get next poster'),
+    slides: Translate.string('Get next slides'),
+  };
+
+  return (
+    <Modal onClose={onClose} closeOnDimmerClick={false} open>
+      <Modal.Header>{titleByType[editableType]}</Modal.Header>
+      <Modal.Content>
+        <div styleName="filetype-list">
+          {fileTypes.map(fileType => {
+            return (
+              <div key={fileType.id} styleName="filetype-list-item">
+                <div>
+                  <strong>{fileType.name}</strong>
+                </div>
+                <div>
+                  {fileType.extensions.length > 1 ? (
+                    <>
+                      <Button.Group size="small">
+                        {fileType.extensions.map(extension => (
+                          <Button
+                            disabled={loading}
+                            content={extension}
+                            key={extension}
+                            onClick={() => updateExtensionFilter(fileType.id, extension)}
+                            color={
+                              Array.isArray(filters[fileType.id]) &&
+                              filters[fileType.id].includes(extension)
+                                ? 'blue'
```

Comment:
You could import Palette from `web/client/js/utils/palette.js` here and use one of those colors in case we want to stick to our palette. Same goes for other color usages here (like orange etc)

Reply:
we cannot use our palette there since those are SUI colors